src/components/GeriLifeTopbar.vue

```diff
@@ -124,21 +109,50 @@ export default {
   data() {
     return {
       getCookie,
+      drawerRight: false,
+      selected: null,
       openProfile: false,
       languageOptions: [
         { label: "English", value: "en" },
         { label: "Suomi", value: "fi" },
       ],
+      menu: [
+        {
+          icon: "fa fa-users",
+          title: this.$i18n.t("mainLayoutNavbar-residentsLink"),
+          route: "/residents",
+        },
+        {
+          icon: "fa fa-heartbeat",
+          title: this.$i18n.t("mainLayoutNavbar-activitesLink"),
+          route: "/activities",
+        },
+        {
+          icon: "fa fa-home",
+          title: this.$i18n.t("mainLayoutNavbar-homesLink"),
+          route: "/homes",
+        },
+        {
+          icon: "fa fa-chart-line",
+          title: this.$i18n.t("mainLayoutNavbar-reportLink"),
+          route: "/report",
+        },
+        {
+          icon: "fa fa-cog",
+          title: this.$i18n.t("mainLayoutNavbar-settingsLink"),
+          route: "/settings",
+        },
+        {
+          icon: "fa fa-user",
+          title: this.$i18n.t("mainLayoutNavbar-profileLink"),
+          isProfileLink: true,
+        },
+      ],
     };
   },
   computed: {
-    isSmOrAbove() {
-      return (
-        this.$q.screen.sm ||
-        this.$q.screen.md ||
-        this.$q.screen.lg ||
-        this.$q.screen.xl
-      );
+    isMdOrAbove() {
+      return this.$q.screen.md || this.$q.screen.lg || this.$q.screen.xl;
```

Comment:
Please use the existing Quasar responsive classes (e.g. `gt-sm`):

https://quasar.dev/style/visibility#window-width-related

> You can also show some DOM element or component if it’s lower than one of the sizes. Same for greater than one of the sizes. Just attach lt- or gt- prefixes, which come from “lower than” and “greater than”. Example: lt-md (display on xs and sm only), lt-xl (display on xs, sm, md and lg windows only), gt-md (display on greater than medium windows: lg and xl).